README.md

```diff
@@ -89,27 +89,14 @@ There are two stages, testing and deployment.
 
 1. **Update the -tools version**: Once the tests pass, update the `setup.py` version and create a tag in the -tools repository at the same SHA you identified earlier.
 
-   - Determine the next [semver](https://packaging.python.org/en/latest/specifications/version-specifiers/#version-specifiers) number as appropriate for the changes in this release. e.g. `1.1.4`
-   - Create a local version identifier, with the format `YYYY-MM-DD-A`, where `YYYY-MM-DD` represents release date, and `-A` is used for the first release of the day (followed by `-B`, `-C`, etc., for subsequent same-day releases). e.g. `2024-08-24-A`
+  - Determine the next [stamp](https://blog.aspect.build/versioning-releases-from-a-monorepo) (which is valid [semver](https://packaging.python.org/en/latest/specifications/version-specifiers/#version-specifiers)) number as appropriate for the changes in this release. e.g. `2024.40.2`.
```

Comment:
This description is rather opaque and the linked articles are long, but I see that it boils down to "run stamp.sh, probably with --release".
But maybe a crisper synopsis would help. Even just the comments in stamp.sh about how the tag is formed are probably enough.

Reply:
I think I wrote this first but didn't get back to updating it.

Done.